scene/main/node.cpp

```diff
@@ -957,6 +957,11 @@ String Node::validate_child_name(Node *p_child) {
 	_generate_serial_child_name(p_child, name);
 	return name;
 }
+
+String Node::validate_child_name(Node *p_child, StringName p_name) {
+	_generate_serial_child_name(p_child, p_name);
+	return p_name;
+}
```

Comment:
I would rename this function to `prevalidate_child_name` to make it clear what is the difference with the other function.